src/types/index.tsx

```diff
@@ -119,3 +119,10 @@ export type MissingShoeReport = {
   shoesToken: string;
   description: string;
 };
+
+export type FundType = {
```

Comment:
I believe if we are  defining this by using type, we may not need to explicitly use FundType.",

Reply:
have improved my interface using the extended method.